src/Components/ProductCard/ProductCard.module.css

```diff
@@ -0,0 +1,60 @@
+.image {
+  width: 100%;
+  flex-grow: 1;
+}
+
+.title {
+  margin: 0.5rem 0.2rem;
+  margin-left: 0.5rem;
+  font-weight: bold;
+  font-size: 0.9rem;
+}
+
+.price {
+  margin: 0.5rem;
+  font-weight: bold;
+}
+
+.productCard {
+  height: 350px;
+  padding: 0.5rem;
+  box-shadow: none;
+  border: 1px solid #0000000e;
+  border-radius: 0;
+  max-width: 300px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  background-color: white;

```

Comment:
don't use color name use RGB or HEX, All browser is not compatible with color name

Reply:
Done.